app/src/test/java/org/oppia/android/app/accessibility/FakeAccessibilityServiceTest.kt

```diff
@@ -0,0 +1,42 @@
+package org.oppia.android.app.accessibility
+
+import android.view.View
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.mockito.Mockito.mock
+import org.oppia.android.util.accessibility.FakeAccessibilityService
+
+/** Tests for [FakeAccessibilityService]. */
+class FakeAccessibilityServiceTest {
+  private var accessibilityManager = FakeAccessibilityService()
```

Comment:
This should be injected instead of constructed (it's more future-proof since we may add injected fields to the constructor in the future).

Reply:
Done